meta-provider/meta-provider/README.md

```diff
@@ -0,0 +1,114 @@
+# Meta provider notes
+
+## BYOH provider
+
+The Bring-Your-Own-Host provider was chosen for this test for two reasons:
+
+1. Due to its design (you provision the host yourself), it is very easy to set up the hosts in various ways to accommodate the test (e.g. use a VM in the same network that the metal3-dev-env uses).
+2. It was mentioned on slack as one of the providers that works when combining multiple providers for a single cluster.
+
+**The test was successful**: It is possible to combine BYOH and metal3 to create a single cluster.
+More details below.
+
+## Test BYOH + Metal3
+
+On a high level this is how it is done:
+
+1. Create a cluster and control plane backed by the BYO provider (but no MachineDeployment).
+2. Create a metal3cluster and related resources with control plane endpoint pointing to the BYO control plane.
+
+This is all it takes to make it "work", but of course there are still some weird things.
+For example the Cluster only has one infrastructureRef and it is pointing to the BYO provider.
```

Comment:
if you can put a yaml here to show that it is not a list

Reply:
or it can be added in the future.

Reply:
Added